python/torch_mlir_e2e_test/linalg_on_tensors_backends/refbackend.py

```diff
@@ -65,6 +65,18 @@ def consume_return_f32(a):
         def consume_return_f64(a):
             self.result = a
 
+        @ctypes.CFUNCTYPE(None, ctypes.POINTER(UnrankedMemRefDescriptor),
+                          ctypes.POINTER(UnrankedMemRefDescriptor))
+        def consume_return_mrf32_mri64(arg0, arg1):
+            self.result = unranked_memref_to_numpy(
+                arg0, np.float32), unranked_memref_to_numpy(
+                    arg1,
+                    np.int64)
+
+        self.ee.register_runtime(
+            "refbackend_consume_func_return_mrf32_mri64",
+            consume_return_mrf32_mri64)
```

Comment:
nit: I think its better to move this after 

https://github.com/llvm/torch-mlir/blob/acb50008a9d50e65b82f473ae8463c36121c969c/python/torch_mlir_e2e_test/linalg_on_tensors_backends/refbackend.py#L110-L111

to keep the organization consistent